MatrixSDK/Data/RoomSummaryStore/CoreData/Models/MXRoomSummaryMO.swift

```diff
@@ -64,6 +64,7 @@ public class MXRoomSummaryMO: NSManagedObject {
     @NSManaged public var s_membersCount: MXRoomMembersCountMO?
     @NSManaged public var s_trust: MXUsersTrustLevelSummaryMO?
     @NSManaged public var s_lastMessage: MXRoomLastMessageMO?
+    @NSManaged public var s_userIdsSharingLiveBeacon: [String]?
```

Comment:
If you want to filter rooms by using this parameter on Core Data level, it may be a good idea to define it as `String` instead, by delimiting with `StringArrayDelimiter` (or any other delimiter according to your needs), like we did for `s_aliases` property. It's very painful to perform filtering on a `Transformable` parameter on db level.